src/Liquidator.sol

```diff
@@ -10,20 +10,20 @@ import { ERC20, SafeTransferLib } from "../lib/solmate/src/utils/SafeTransferLib
 import { IAccount } from "./interfaces/IAccount.sol";
 import { ILendingPool } from "./interfaces/ILendingPool.sol";
 import { Owned } from "lib/solmate/src/auth/Owned.sol";
+import { ILiquidator } from "./interfaces/ILiquidator.sol";
+import { RiskModule } from "lib/accounts-v2/src/RiskModule.sol";
+import { SafeCastLib } from "../lib/solmate/src/utils/SafeCastLib.sol";
 
```

Comment:
contract natspec

Reply:
IFactory, ILiquidator and SafeCastLib are not used. Would use ILiquidator though as in `contract Liquidator is ILiquidator`